scale/docs/rest/v6/scale_file.rst

```diff
@@ -0,0 +1,418 @@
+
+.. _rest_v6_scale_file:
+
+v6 Scale File Services
+========================================================================================================================
+
+These services provide access to information about general files that are being tracked by Scale.
+
+.. _rest_v6_scale_file_list:
+
+v6 Scale Files
+----------------------
+
+**Example GET /v6/files/ API call**
+
+Request: GET http://.../v6/files/
+
+Response: 200 OK
+
+ .. code-block:: javascript  
+    { 
+        "count": 55, 
+        "next": null, 
+        "previous": null, 
+        "results": [ 
+            { 
+                "id": 465, 
+                "workspace": { 
+                    "id": 2, 
+                    "name": "Products" 
+                }, 
+                "file_name": "my_file.kml", 
+                "media_type": "application/vnd.google-earth.kml+xml", 
+                "file_size": 100, 
+                "data_type": [], 
+                "is_deleted": false, 
+                "url": "http://host.com/file/path/my_file.kml", 
+                "created": "1970-01-01T00:00:00Z", 
+                "deleted": null, 
+                "data_started": null, 
+                "data_ended": null, 
```

Comment:
Put in the default `"1970-01-01T00:00:00Z"` for `data_started` and `data_ended`.